docs/clients/pulsing_rectangular_input_parser.rst

```diff
@@ -110,10 +110,13 @@ Type ID 89 | Pulsing Rectangular Input Parser
         :widths: 8 18 15 10 5 50
         :class: tight-table
 
-        +--------+-----------------------+----------------+-----------+----------------------+---------------------------------------------------------------------+
-        | Sub ID | Short Name            | Access         | Data Type | Unit                 | Note                                                                |
-        +========+=======================+================+===========+======================+=====================================================================+
-        | 0      | pulsing_voltage_mode  | get, set, save | uint8     | :math:`\text{Enum}`  | 0 = Supply Voltage Mode, 1 = Voltage Limit Mode                     |
-        +--------+-----------------------+----------------+-----------+----------------------+---------------------------------------------------------------------+
-        | 1      | pulsing_voltage_limit | get, set, save | float     | :math:`V`            | This sets the max pulsing voltage limit when in Voltage Limit Mode. |
-        +--------+-----------------------+----------------+-----------+----------------------+---------------------------------------------------------------------+
+        +--------+-----------------------+----------------+-----------+----------------------+--------------------------------------------------------------------------------------------+
+        | Sub ID | Short Name            | Access         | Data Type | Unit                 | Note                                                                                       |
+        +========+=======================+================+===========+======================+============================================================================================+
+        | 0      | pulsing_scaling_mode  | get, set, save | uint8     | :math:`\text{Enum}`  | 0 = Supply Voltage Mode (voltage pulsing only), 1 = Scaling Limit Mode                     |
```

Comment:
Maybe worth mentioning that on velocity pulsing firmware we make this entry read-only. May just be useful to make that clear so people aren't surprised by it.

Relevant code bit here: https://bitbucket.org/iqcontrol/common_high_level_files/src/97207020f24b5b2627e756d16888e2036f552af0/COMMON_CPP/src/pulsing_rectangular_input_parser.cpp#lines-27